stubs/app/Actions/Jetstream/DeleteUser.php

```diff
@@ -14,6 +14,7 @@ class DeleteUser implements DeletesUsers
      */
     public function delete($user)
     {
+        $user->deleteProfilePhoto();
```

Comment:
```suggestion
        if (\Laravel\Jetstream\Features::managesProfilePhotos()) {
                $user->deleteProfilePhoto();
       }
```

Reply:
Ohhh right, yeah just came across that feature check yesterday but didn't quite connect the dots here. Added it!